src/popup/hooks/index.ts

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from "react";
+import { loadApiKey, saveApiKey, clearApiKey } from "../../storage";
+
+const API_HOST =
+  process.env.NODE_ENV === "production"
+    ? "https://api.growthbook.io"
+    : "http://localhost:3100";
```

Comment:
We can use these values as defaults, but we need to let the user override this.  When self-hosting GrowthBook, the API will be deployed on a custom URL, like `https://growthbook-api.example.com`